pkgs/applications/misc/bambu-studio/default.nix

```diff
@@ -166,6 +166,16 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  # needed to prevent collisions between the LICENSE.txt files of
+  # bambu-studio and orca-sliver.
+  postInstall = ''
+    mkdir -p $out/share/doc
+    mv $out/LICENSE.txt $out/share/doc/LICENSE.txt
+    if [ -f $out/README.md ]; then
```

Comment:
is there a situation when README.md not there ?

i only looked at babmu-studio

Reply:
> is there a situation when README.md not there ?
> 
> i only looked at babmu-studio

Yes: with orca-slicer.